js/repl/ReplOptions.js

```diff
@@ -501,6 +501,19 @@ class ExpandedContainer extends Component<Props, State> {
                 >
                   Built-ins
                 </LinkToDocs>
+                <select
+                  value={envConfig.corejs}
+                  className={styles.envPresetSelect}
+                  onChange={this._onEnvPresetSettingChange("corejs")}
+                  disabled={
+                    !envConfig.isEnvPresetEnabled ||
+                    !envConfig.isBuiltInsEnabled ||
+                    runtimePolyfillState.isEnabled
+                  }
+                >
+                  <option value="2">core-js 2</option>
+                  <option value="3.6">core-js 3</option>
```

Comment:
I think that the UI should show 3.6, to make it clearer that the corresponding option is `corejs: "3.6"` and not `corejs: 3`.

Reply:
changed.